.circleci/config.yml

```diff
@@ -22,6 +24,19 @@ jobs:
 
       # run tests!
       - run: npm test
+
+      # generate badges 
+      - run : 
+          name : "Generate Code Coverage and Badges"
+          command : npm run test:badges
```

Comment:
Can this be removed as you are setting up coveralls?

Reply:
Yes I'll be removing it.. Doing some final tests on my fork. I'll keep you updated. I might need your help to setup coveralls for this repo. 
Also please note that coveralls provides one aggregated coverage badge and not 4 separata badges